src/vsr/replica.zig

```diff
@@ -4032,6 +4116,7 @@ pub fn ReplicaType(
                 self.commit_journal(self.commit_max);
```

Comment:
This line creates a sort of loop — committing can trigger a checkpoint, which may advance `op_repair_max`, which would cause us to go through `if (self.op < self.op_repair_max()) {` again.